tests/unit/test_handler.py

```diff
@@ -55,19 +54,56 @@ def apigw_event():
             "CloudFront-Forwarded-Proto": "https",
             "Accept-Encoding": "gzip, deflate, sdch",
         },
-        "pathParameters": {"proxy": "/examplepath"},
+        "pathParameters": {"proxy": path},
         "httpMethod": "POST",
         "stageVariables": {"baz": "qux"},
-        "path": "/examplepath",
+        "path": path,
     }
 
 
-def test_lambda_handler(apigw_event, mocker):
+@pytest.fixture()
+def all_accounts_event():
+    return apigw_event('/all/accounts.json')
```

Comment:
didn't this change to `costcenters.json`?  please update it in the image as well.

Reply:
the value used in test is completely arbitrary because we've mocked everything outside of the handler function, but I'll make sure the image(s) are updated